blockstore/splitstore/splitstore.go

```diff
@@ -676,6 +706,7 @@ func (s *SplitStore) Close() error {
 		}
 	}
 
+	s.reifyCond.Broadcast()
```

Comment:
Looks like we're not waiting for reify workers to finish here.. but that's fiine I guess?

Reply:
yeah good point.  It is pribably ok, but it will be better to wait for them.

Reply:
I added a wait group for the reify workers and wait for them on close.